changelog/unreleased/3177

```diff
@@ -1,6 +1,6 @@
 Enhancement: Align previews actions
 
-We've aligned the behaviour for every preview fragment. Images, videos, audios and texts show the same actions now.
+Behaviour was alligned through every preview fragment. Images, videos, audios and texts show the same actions now.
```

Comment:
Same typo here

Reply:
yes, typo es de same, copied by calens from unreleased to changelog. So, also fixed :)